### ISSUE TITLE
Add the full database of LINDDUN threats to Threat Dragon

### ISSUE THREAD

**Issue by rhite-tech** [OWASP/threat-dragon#797]
**Describe what problem your feature request solves**:
LINDDUN contains a database of 35 threats. Threat Dragon already has support for the categories, similar to the other frameworks, but it would be nice if the threats under each category would become available from a drop-down menu as well.
The way threats are organized under the categories can be observed here: https://downloads.linddun.org/linddun-go/default/v20230802/go.pdf

**Describe the solution you'd like**:
When you add a threat (or edit one), you can already select the threat type, which corresponds to the linddun category.
If the linddun cards were to be digitalized in json format - similar to plot4ai - then this could serve as a database; then, based on this json file and the selected type/category, the relevant threats could be loaded and used to populate a drop-down box called 'threat'. The user could then select the applicable threat.
Additionally, the rest of the information on the linddun cards could be used to prefill the Description-field in Threat Dragon.

**Additional context**:
LINDDUN is very similar to PLOT4ai, which was also recently added to Threat Dragon. Adding this functionality would also open the possibility to add the threats from PLOT4ai to threat dragon.

**Comment by jgadsden** [comment]
another very good suggestion @rhite-tech , did you want to work on this in parallel with #796 ?

**Comment by bitnesswise** [comment]
Yes, my expectation is that once the code for one model is in place it will be very easy to add the same functionality for the other.

Maybe good to know: I've also reached out to the team behind LINDDUN to ask if they like to be involved.

**Comment by rhite-tech** [comment]
Yes, my expectation is that once the code for one model is in place it will be very easy to add the same functionality for the other.

Maybe good to know: I've also reached out to the team behind LINDDUN to ask if they like to be involved.

**Comment by raghav1030** [comment]
Hey @jgadsden, could you please assign this task to me? I'm ready to start working on it.

**Comment by jgadsden** [comment]
@rhite-tech are you still working on this issue?

**Comment by jgadsden** [comment]
This is in active development by @rhite-tech
